src/api/uploads/MultiputUpload.js

```diff
@@ -655,6 +639,23 @@ class MultiputUpload extends BaseMultiput {
      */
     partUploadErrorHandler = (error: Error, eventInfo: string): void => {
         this.sessionErrorHandler(error, LOG_EVENT_TYPE_PART_UPLOAD_RETRIES_EXCEEDED, eventInfo);
```

Comment:
It seems like if we pause, we shouldn't be calling this, right?

Reply:
Not sure about this one, but I can play around with it. I think we still need to call `sessionErrorHandler` because it calls the errorCallback which is set to be `ContentUploader.handleUploadError` from `resume` in `ContentUploader.resumeFile`. The error callback updates the Uploads Manager in EUA to show uploads error.

Reply:
Ok tried it and we definitely need it otherwise it gets into an infinite loading state.